$NetBSD$

One of the three patches replacing the distribution patch
under work/nbitools-R6.3/config/md.patch

--- config/makedepend/parse.c.orig	1997-08-22 08:21:37.000000000 +0000
+++ config/makedepend/parse.c
@@ -1,4 +1,5 @@
 /* $XConsortium: parse.c /main/33 1996/12/04 10:11:28 swick $ */
+
 /*
 
 Copyright (c) 1993, 1994  X Consortium
@@ -40,7 +41,7 @@ gobble(filep, file, file_red)
 	register char	*line;
 	register int	type;
 
-	while (line = getline(filep)) {
+	while (line = mdgetline(filep)) {
 		switch(type = deftype(line, filep, file_red, file, FALSE)) {
 		case IF:
 		case IFFALSE:
@@ -529,7 +530,7 @@ find_includes(filep, file, file_red, rec
 	register int	type;
 	boolean recfailOK;
 
-	while (line = getline(filep)) {
+	while (line = mdgetline(filep)) {
 		switch(type = deftype(line, filep, file_red, file, TRUE)) {
 		case IF:
 		doif:
